src/components/app/ListOfPublishedProfiles.js

```diff
@@ -146,6 +155,26 @@ class PublishedProfile extends React.PureComponent<
     this.setState({ confirmDialogIsOpen: true });
   };
 
+  preventClick(e) {
+    e.preventDefault();
+  }
+
+  _renderPossibleErrorMessage() {
+    const { error } = this.state;
+    if (!error) {
+      return null;
+    }
+
+    return (
+      <p className="publishedProfilesDeleteError">
+        An error happened while deleting this profile.{' '}
+        <a href="#" title={error.message} onClick={this.preventClick}>
+          Hover to know more.
+        </a>
+      </p>
+    );
+  }
```

Comment:
Because this is hard for you to check how this looks, here is a screenshot:
![image](https://user-images.githubusercontent.com/454175/93912782-79992e80-fd04-11ea-927f-6cf5de1d5fe3.png)
I tried using a "help" icon instead of this link, but this didn't look good in my opinion. Happy to hear more suggestion though. Also I'm not so sure of the accessibility, but I'd say this is isn't bad.

Reply:
It's actually pretty easy to reproduce if you have 2 windows open, and try to delete them in both. I filed a "good first issue" #2810 that would help mitigate that.

<img width="748" alt="image" src="https://user-images.githubusercontent.com/1588648/94451671-4b887400-0174-11eb-80d2-a084840b9e11.png">

I'm not a huge fan of the hover solution, I think it would be simpler on the user to dump the information directly onto the screen. In fact, some of the errors we could give more human-readable messages. `404` means that we know this profile was already deleted, and we should update the view of the interface. Of course, #2810 would mitigate a lot of these issues.